src/esm_runscripts/helpers.py

```diff
@@ -136,6 +137,78 @@ def assemble_log_message(
     return line
 
 
+def update_reusable_files(config, reusable_filetypes=None):
+    """
+    Reusable files, in general (``config["general"]["reusable_filetypes"]``), or model
+    specific (``config[<model>]["reusable_filetypes"]``) forces files of a given type,
+    such as ``bin``, ``input``, ``forcing``, etc., to be copied only once into the
+    general experiment folder, and then **reused** them through out every run. If a
+    given file type is not present in this list, that file type will be copied from
+    its external source (i.e. for binaries from the folder where the model is compiled).
+
+    This method takes care of removing the reusable file types from
+    ``reusable_filetypes``, specified by the user in the ``esm_runscripts`` call with
+    the ``--update-files`` flag.
+
+    Parameters
+    ----------
+    config : dict
+        Dictionary containing the information about the experiment.
+    reusable_filetypes : list
+        List of reusable file types that can be specified by the user.
+
+    Returns
+    -------
+    config : dict
+        If the user gives no ``reusable_filetypes`` as input. ``config`` contains the
+        changes requested by the user with the ``--update-flag``.
+    reusable_filetypes : list
+        If the user gives ``reusable_filetypes`` as input. ``reusable_filetypes``
+        contains the changes requested by the user with the ``--update-flag``.
```

Comment:
Only a minor thing: check in Sphinx just to be sure, it might be that this is documented as returning two things. If that is the case, you could try:

```suggestion
    dict or list
        If the user gives no ``reusable_filetypes`` as input. A dict 
        ``config`` is returned, which contains the changes requested 
        by the user with the ``--update-flag``.
        
        If the user gives ``reusable_filetypes`` as input, a list is 
        returned. This list ``reusable_filetypes`` contains the changes
        requested by the user with the ``--update-flag``.
```